package.json

```diff
@@ -35,10 +35,11 @@
   "homepage": "https://skygear.io/",
   "dependencies": {
     "@babel/polyfill": "^7.4.4",
-    "@skygear/node-client": "^2.0.0-alpha.11",
+    "@skygear/node-client": "^2.0.0-alpha.13",
     "@zeit/dockerignore": "0.0.5",
     "chalk": "^2.4.2",
     "cli-table3": "^0.5.1",
+    "env-paths": "2.2.0",
```

Comment:
I found that `env-paths` changed the config path of mac to ` /Users/{user}/Library/Preferences/`.
I think it make sense for normal app, but not for command line program.
It is better if we can its linux version path in mac too, see if we want to implement ourselves?

Reply:
I think it make sense? We can also say the same thing for Windows platform, but I think following platform's convention on config location is better.

Reply:
The conventional configuration location of command line program is universal for UNIX-like system.